src/dvc_task/app/filesystem.py

```diff
@@ -121,7 +113,7 @@ def _iter_folder(
             queue: Optional name of queue.
         """
         with self.connection_for_read() as conn:  # type: ignore[attr-defined]
-            with conn.channel() as channel:
+            with conn.channel() as channel:  # type: ignore[attr-defined]
```

Comment:
celery-types does not have type stub for `Channel` as a contextmanager.

https://github.com/sbdchd/celery-types/blob/60bbd4361297391d1853d1d0a4b720573521df58/kombu-stubs/transport/base.pyi#L7